config/config_helpers.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 var INTERPOLATION_SYNTAX_REGEX = regexp.MustCompile("\\$\\{.*?\\}")
-var HELPER_FUNCTION_SYNTAX_REGEX = regexp.MustCompile("\\$\\{(.*?)\\(\\)\\}")
+var HELPER_FUNCTION_SYNTAX_REGEX = regexp.MustCompile(`\$\{(.*?)\((.*?)\)\}`)
+var HELPER_FUNCTION_GET_ENV_PARAMETERS_SYNTAX_REGEX = regexp.MustCompile(`\"(?P<ENV>[a-zA-Z_]+[a-zA-Z0-9_]+?)\"\,\s*\"(?P<default>.*?)\"`)
```

Comment:
Regarding the expression, I confess I used as reference: http://stackoverflow.com/questions/2821043/allowed-characters-in-linux-environment-variable-names?answertab=votes#tab-top and totally forgot about the windows case.

I avoided the `.` since it seems a bit to aggressive even with `go` ignoring new lines. What about `(?P<env>[^=]+?)` that excludes `=` that is not allowed in any system (*nix and windows) and could lead to confusion.

I've added a few more tests to cover edge cases.